<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceLatLng</class>
 <widget class="QDialog" name="DistanceLatLng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog.bgw{
	border-right:1px solid rgb(170,170,170);
	background:qlineargradient(spread:reflect,
		x1:0,y1:0,x2:1,y2:0,
		stop:0 rgba(207,210,184,255),
		stop:1 rgba(198,210,100,255));
}
QPushButton:pressed {
         background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #aadbfe, stop: 1 #f6f7fa);

}</string>
  </property>
  <property name="class" stdset="0">
   <string>bgw</string>
  </property>
  <widget class="QPlainTextEdit" name="polylines">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>计算距离长度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="result">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="result_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>谷歌/高德 偏移坐标转真实坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="result_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.经纬度两点之间距离计算/多点轨迹长度/轮廓长度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.地图坐标偏移到物理坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>55</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;font color='#ff0000'&gt;格式:120.3784822,36.10502583;120.3786108,36.10492528&lt;/font&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
